src/naf-dialog-adapter.js

```diff
@@ -173,6 +249,11 @@ export default class DialogAdapter {
     });
 
     this._protoo.on("notification", notification => {
+      this.scene.emit("rtc_event", {
+        level: "debug",
+        tag: "Signaling",
+        msg: `Notification [${notification.method}]: ${JSON.stringify(notification.data)}`
+      });
```

Comment:
I don't have a good sense for how frequently all of these events will be emitted, but if they happen constantly, I'd be worried about the performance impact and garbage collection due to all of these objects, strings and the events themselves. 

Ideally there would be no additional impact on performance or memory usage when the debug panel is closed. 

Perhaps these emits should be encapsulated in a helper function that can be turned off with a global flag?
Something like:
```js
function emitRTCEvent(level, tag, msgFunc) {
  if (!window.APP.store.state.preferences.showRtcDebugPanel) return;
  this.scene.emit("rtc_event", { level, tag, msg: msgFunc() });
} 

...

emitRTCEvent(
  "debug", 
  "Signaling",
  () => `Notification [${notification.method}]: ${JSON.stringify(notification.data)}`
);
```

Reply:
That sounds about right, I'll update with that.